integration-tests/utils/integration-utils.js

```diff
@@ -75,7 +75,7 @@ const getTableColIndex = async (page, { tableName, columnHeader }) => {
 const getTableCellByContents = async (page, { tableName, columnHeader, textContains, isDescendant = false }) => {
```

Comment:
Given that this is no longer a contains, let's rename this to `text`
```suggestion
const getTableCellByContents = async (page, { tableName, columnHeader, text, isDescendant = false }) => {
```